### PR TITLE
Delete configmap if a deploy remove the nginx

### PR DESCRIPTION
**Pull request by yagonobre**

### CODE REVIEW

File: pkg/server/deploy/deploy.go [hunk 1]
```diff
@@ -150,6 +152,12 @@ func (ops *DeployOperations) createOrUpdateDeploy(a *app.App, confFiles *DeployC
 			log.WithError(err).Errorf("Creating config to nginx of app %s", a.Name)
 			return
 		}
+	} else {
+		err := ops.k8s.DeleteConfigMap(a.Name, a.Name)
+		if !ops.k8s.IsNotFound(err) {
```
Comment (aguerra) [inline pkg/server/deploy/deploy.go]: It will return a nil error to the channel, right?

Comment (yagonobre) [reply pkg/server/deploy/deploy.go]: It will return the error if it is not `not found`

Comment (yagonobre) [reply pkg/server/deploy/deploy.go]: yes, you're right!

Comment (yagonobre) [reply pkg/server/deploy/deploy.go]: Updated!

**Comment by drgarcia1986** [comment]
Could you update changelog after rebase?

**Pull request merged**